docs/content/en/docs/how-tos/builders/_index.md

```diff
@@ -191,22 +193,17 @@ each produce a separate container image.
 To build a Maven multi-module project, first identify the modules that should
 produce a container image. Then for each such module:
 
-  1. Create a Skaffold `artifact` in the `skaffold.yaml`:
-     - Set the `artifact`'s `context` field to the root project location.
-     - Add a `jibMaven` element and set its `module` field to the module's
-       `:artifactId`, `groupId:artifactId`, or the relative path to the module
-       _within the project_.
-  2. Configure the module's `pom.xml` to bind either `jib:build` or `jib:dockerBuild` to
-     the `package` phase as appropriate (see below).
-
-This second step is necessary at the moment as Maven applies plugin goals specified
-on the command-line, like `jib:build` or, to all modules and not just the modules
-producing container images.
-The situation is further complicated as Skaffold speeds up deploys to a local cluster,
-such as `minikube`, by building and loading container images directly to the
-local cluster's docker daemon (via `jib:dockerBuild` instead of `jib:build`),
-thus saving a push and a pull of the image.
-We plan to improve this situation [(#1876)](https://github.com/GoogleContainerTools/skaffold/issues/1876).
+  - Create a Skaffold `artifact` in the `skaffold.yaml`:
+  - Set the `artifact`'s `context` field to the root project location.
+  - Add a `jibMaven` element and set its `module` field to the module's
+    `:artifactId`, `groupId:artifactId`, or the relative path to the module
+    _within the project_.
+
+{{% alert title="Updating from earlier versions" %}}
```

Comment:
I don't think this is rendering? At least not in the preview, is this some markdown extension?

Reply:
It's a Hugo shortcode.

Reply:
oh, where do we expect this to render to?

Reply:
<img width="727" alt="Screen Shot 2019-06-10 at 3 19 47 PM" src="https://user-images.githubusercontent.com/202851/59220524-32c90900-8b93-11e9-91db-92ad8f7b975c.png">

Reply:
https://skaffold.dev/docs/how-tos/builders/